src/main/kotlin/com/redhat/devtools/intellij/kubernetes/tree/KubernetesDescriptors.kt

```diff
@@ -126,8 +129,28 @@ object KubernetesDescriptors {
 		model,
 		project
 	) {
+		private val df = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'")
+		private val trans = listOf(1000, 60, 60, 24)
+		private val units = listOf("s", "m", "h", "d")
 		override fun getLabel(element: Pod): String {
-			return element.metadata.name ?: "unknown"
+			if (null != element.metadata.name) {
+				var started = 0L
+				try {
+					val startedAt = df.parse(element.status.startTime).time
+					started = System.currentTimeMillis() - startedAt - TimeZone.getDefault().rawOffset
+				} catch (_: Throwable) {}
+				var readable: Long = 0
+				var i = 0
+				while (i < trans.size) {
+					started /= trans[i]
+					if (started <= 0) break
+					readable = started
+					++i
+				}
+				if (0 < i) --i
+				return "[" + element.status.phase.substring(0, 4) + "-" + readable + units[i] + "]" + element.metadata.name
```

Comment:
In a birds view I fear that users would not know what the additional information (ex `[Run-12d] mypod`) is all about (that was my case before looking into the code). 
Neither vscode nor the jetbrains tooling does this. IMHO we should either find a way to better explain this info or maybe move it to the properties instead (where there's the state and IP currently). 
<img width="258" alt="image" src="https://user-images.githubusercontent.com/25126/216087305-8b9e7860-167b-4ee6-a6cf-5aef1295f4a4.png">
It would also be better to move this label to the "decorations" like we currently do for the CRD. The additional information is better separated from the name:
<img width="258" alt="image" src="https://user-images.githubusercontent.com/25126/216093271-4726e296-9e41-47b0-8f42-c12c198e6bdf.png">

Reply:
btw. kotlin would prefer to do 
```kotlin
"[" + element.status.phase.substring(0, 4) + "-" + readable + units[i] + "]" + element.metadata.name
```
with the string template functionality
```
"[ ${element.status.phase.substring(0, 4)}-$readable${units[i]}] ${element.metadata.name}"
```